src/uikit.cs

```diff
@@ -700,6 +700,18 @@ public partial interface NSTextStorage {
 		[Since (7,0)]
 		[Notification, Internal, Field ("NSTextStorageDidProcessEditingNotification")]
 		NSString DidProcessEditingNotification { get; }
+
+		// we need to hide this methods inherited from the NSMuttableAttributedString since the class does not implement them
+#if XAMCORE_4_0
+		[New, Abstract]
```

Comment:
Adding manual code for `XAMCORE_4_0` is not worth the time. Simply add an `#error` inside the block so we'll see the comment when the time comes. By then our generator tooling might be quite different...